geoh5py/data/visual_parameters.py

```diff
@@ -110,6 +110,30 @@ def colour(self, rgb: list | tuple | np.ndarray):
 
         self.set_tag("Colour", str(value))
 
+    @property
+    def filter_basement(self) -> None | float:
+        """
+        Filter basement tag for VPmodel.
+        """
+        element = self.get_tag("Filterbasement")
+
+        if element is None or not element.text:
```

Comment:
are we sure `element.text` is always float?

Reply:
Currently, VisualParameters is agnostic of the parent object. So if `element` is None, that means it's not a VPmodel, so no we don't want to set it.